code/src/test/java/com/expediagroup/sdk/core/http/ResponseJavaTest.java

```diff
@@ -0,0 +1,24 @@
+package com.expediagroup.sdk.core.http;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ResponseJavaTest {
+    @Test
+    @DisplayName("should access default builder static method")
+    public void defaultStaticBuilderIsAccessibleFromJava() throws IOException {
```

Comment:
We have a test which throws an exception?

Reply:
We either add checked exception to the test signature or wrap the test with try/catch. Java! :)

This test is just to ensure Java compatibility, no real logic